Schema/V1/metadatakatalog.xsd

```diff
@@ -960,6 +960,92 @@
     </xs:complexContent>
   </xs:complexType>
 
+  <xs:complexType name="representerer">
+    <xs:annotation>
+      <xs:documentation>M374</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Ny datatype</xs:documentation>
+    </xs:annotation>
+    <xs:complexContent>
+      <xs:extension base="identifikatorer"/>
+    </xs:complexContent>
+  </xs:complexType>
+
+  <xs:complexType name="moetesaksBeskrivelse">
+    <xs:annotation>
+      <xs:documentation>M375</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Nytt felt. Feltet tekstFormat er av typen kode med kodelisten:  no.ks.fiks.arkiv.v1.kodelister.tekstformat</xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="tekstInnhold" type="xs:string"/>
+      <xs:element name="tekstFormat" type="kode"/>
+    </xs:sequence>    
+  </xs:complexType>
+
+  <xs:complexType name="behandlingsprotokoll">
+    <xs:annotation>
+      <xs:documentation>M376</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Nytt felt</xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="tekstInnhold" type="xs:string"/>
+      <xs:element name="tekstFormat" type="kode"/>
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:simpleType name="moeteStemme" final="restriction" >
+    <xs:annotation>
+      <xs:documentation>M377</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Nytt felt. Feltet tekstFormat er av typen kode med kodelisten: </xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Ja" />
+      <xs:enumeration value="Nei" />
+      <xs:enumeration value="Blankt" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:complexType name="moetedeltaker">
+    <xs:annotation>
+      <xs:documentation>M378</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Nytt felt. Feltet tekstFormat er av typen kode med kodelisten: </xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="navn" type="moetedeltakerNavn"/>
```

Comment:
Bruke identifikatorer i stedet for navn og personid?

Reply:
Done